microsoft/testsuites/performance/storageperf.py

```diff
@@ -125,6 +133,199 @@ def perf_premium_datadisks_io(self, node: Node, environment: Environment) -> Non
             max_iodepth=64,
         )
 
+    @TestCaseMetadata(
+        description="""
+        This test case uses fio to test performance of nfs server over TCP with
+        VM's initialized with SRIOV network interface.
+        """,
+        priority=2,
```

Comment:
The priority should plus 1. Most perf test cases should be P3. If it runs faster (in 5 minutes), and it's important, we can move some to p2.